LICENSE-CODE

```diff
@@ -0,0 +1,17 @@
+The MIT License (MIT)
```

Comment:
Why 2 separate files called LICENSE and LICENSE-CODE? What's the reason for making license related changes

Reply:
removed LICENSE-CODE